File: developer/Samples/NET 4.6/WF_WCF_Samples/WCF/Basic/Binding/ByteStreamEncoder/VB/ByteStreamHttpBindingServer/IHttpHandler.vb
'----------------------------------------------------------------
' Copyright (c) Microsoft Corporation.  All rights reserved.
'----------------------------------------------------------------

Imports Microsoft.VisualBasic
Imports System.ServiceModel
Imports System.ServiceModel.Channels

Namespace Microsoft.Samples.ByteStreamEncoder

    <ServiceContract()> _
    Friend Interface IHttpHandler
        <OperationContract(Action:="*", ReplyAction:="*")> _
        Function ProcessRequest(ByVal request As Message) As Message
    End Interface
End Namespace

File: developer/Samples/NET 4.6/WF_WCF_Samples/WCF/Basic/Binding/ByteStreamEncoder/CS/ByteStreamHttpBindingServer/IHttpHandler.cs
//----------------------------------------------------------------
// Copyright (c) Microsoft Corporation.  All rights reserved.
//----------------------------------------------------------------

using System.ServiceModel;
using System.ServiceModel.Channels;

namespace Microsoft.Samples.ByteStreamEncoder
{

    [ServiceContract]
    interface IHttpHandler
    {
        [OperationContract(Action = "*", ReplyAction = "*")]
        Message ProcessRequest(Message request);
    }
}
